Automattic/wp-calypso
# Checkout Modal: Fix styles

Related to #

## Proposed Changes

* Fix styles of the checkout modal

| Before | After |
| - | - |
| ![image](https://github.com/Automattic/wp-calypso/assets/13596067/1c9c2198-e15e-40da-88ee-e54d581e0aa6) | ![image](https://github.com/Automattic/wp-calypso/assets/13596067/6c43f01d-5035-4612-b657-eea3caaa5597) |

## Testing Instructions



* Go to Assembler on your free site
* Select Colors & Fonts
* Continue to checkout
* Make sure the styles look good

## Pre-merge Checklist
